boards/BMS/Src/Io/Io_LTC6813.c

```diff
@@ -0,0 +1,374 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "Io_LTC6813.h"
+#include "Io_SharedSpi.h"
+#include "configs/Io_LTC6813Configs.h"
+
+enum CellVoltageRegisterGroups
+{
+    CELL_VOLTAGE_REGISTER_GROUP_A = 0,
+    CELL_VOLTAGE_REGISTER_GROUP_B,
+    CELL_VOLTAGE_REGISTER_GROUP_C,
+    CELL_VOLTAGE_REGISTER_GROUP_D,
+    CELL_VOLTAGE_REGISTER_GROUP_E,
+    CELL_VOLTAGE_REGISTER_GROUP_F,
+    NUM_OF_CELL_VOLTAGE_REGISTER_GROUPS
+};
+
+struct LookupTables
+{
+    // Commands used to read cell voltage register groups.
+    uint16_t cell_voltage_register_group_commands
+        [NUM_OF_CELL_VOLTAGE_REGISTER_GROUPS];
```

Comment:
These are `uint16_t`'s, how should they be interpreted as commands?

Reply:
This is used in `Io_LTC6813_ReadAllCellRegisterGroups`. Basically a command that is used to read from a cell voltage register group. `cell_register_group_cmd` is split into two bytes and transmitted to the IC.

Reply:
It's probably worth having a comment saying basically this here so that people in the future can understand.